android/app/src/main/AndroidManifest.xml

```diff
@@ -25,6 +24,14 @@
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
     <uses-permission android:name="android.permission.READ_CONTACTS" />  
     <uses-permission android:name="android.permission.WRITE_CONTACTS" />  
+        <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Fix indentation please :D

Reply:
no, it is belong to android project, not the same with flutter